android-sdk/src/main/java/com/mobilecoin/lib/TxOutMemo.java

```diff
@@ -0,0 +1,37 @@
+package com.mobilecoin.lib;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import androidx.annotation.NonNull;
+
+/** Represents a TxOut memo. */
+public abstract class TxOutMemo extends Native implements Parcelable {
+
+  @NonNull
+  protected final TxOutMemoType memoType;
+  protected transient boolean validated = false;
```

Comment:
Transient keyword added here for documentation purposes, even though it doesn't have an effect on our new serialization. Putting this keyword here conveys that if a TxOutMemo is serialized and then deserialized, it will be considered invalid until verified, regardless of whether or not it was already validated before serialization. We could save this part of the state if we want to, but this functionality is consistent with the original RTH code before I made changes for serialization.